cmd/version_test.go

```diff
@@ -54,3 +55,39 @@ func TestVersionTooManyArgs(t *testing.T) {
 		t.Fatal("Correct error message not given.")
 	}
 }
+
+func TestOverrideControllerVersion(t *testing.T) {
+
+	sandbox, cleanup := cmdtest.TestSetupWithSandbox(t, true)
+	defer cleanup()
+
+	os.Setenv("APPSODY_CONTROLLER_VERSION", "0.3.0")
+
+	args := []string{"version"}
+	output, err := cmdtest.RunAppsody(sandbox, args...)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if !strings.Contains(output, cmd.VERSION) && !strings.Contains(output, "0.3.0") {
+		t.Fatal("Output does not contain CLI or altered controller version")
+	}
+}
+
+func TestOverrideControllerImage(t *testing.T) {
+
+	sandbox, cleanup := cmdtest.TestSetupWithSandbox(t, true)
+	defer cleanup()
+
+	os.Setenv("APPSODY_CONTROLLER_IMAGE", "appsody/new-controller:0.1.1")
+
+	args := []string{"version"}
+	output, err := cmdtest.RunAppsody(sandbox, args...)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if !strings.Contains(output, cmd.VERSION) && !strings.Contains(output, "appsody/new-controller:0.1.1") {
```

Comment:
wouldn't we want `0.1.1`? Since we split by `:` and take the version?

Reply:
Yeah, testing that because there's a `Log.Info.log` that prints out that it is using the new controller image.  I should test that we get `appsody-controller 0.1.1` too